### ISSUE TITLE
Highlight doc_strings based on content_type

### ISSUE THREAD

**Issue by gogiel** [rmpestano/cukedoctor#118]
Cukedoctor should automatically highlight doc_strings based on their content_type.

Example:
Feature definition:
```cucumber
Feature: test
  Scenario: test
    Given code
    """ruby
    class TestClass
      attr_reader :test
    end
    """
```

JSON
```json
[
  {
    "uri": "features/test.feature",
    "id": "test",
    "keyword": "Feature",
    "name": "test",
    "description": "",
    "line": 1,
    "elements": [
      {
        "id": "test;test",
        "keyword": "Scenario",
        "name": "test",
        "description": "",
        "line": 2,
        "type": "scenario",
        "steps": [
          {
            "keyword": "Given ",
            "name": "code",
            "line": 3,
            "doc_string": {
              "value": "class TestClass\n  attr_reader :test\nend",
              "content_type": "ruby",
              "line": 4
            },
            "match": {
              "location": "features/test.feature:3"
            },
            "result": {
              "status": "undefined"
            }
          }
        ]
      }
    ]
  }
]
```

Based on `"content_type": "ruby"` it should be highlighted as Ruby.

**Comment by rmpestano** [comment]
Hi @gogiel, interesting idea and may be quite easy to implement. Thanks.

**Comment by rmpestano** [comment]
Fixed by #123 and available in `v2.0`
